src/AsymptoticLimits.cc

```diff
@@ -76,8 +76,8 @@ void AsymptoticLimits::applyOptions(const boost::program_options::variables_map
     noFitAsimov_ = vm.count("noFitAsimov") || vm.count("bypassFrequentistFit"); // aslo pick up base option from combine
 
     if (rule_=="CLs") doCLs_ = true;
-    else if (rule_=="CLsplusb") doCLs_ = false;
-    else throw std::invalid_argument("AsymptoticLimits: Rule must be either 'CLs' or 'CLsplusb'");
+    else if (rule_=="Pmu") doCLs_ = false;
+    else throw std::invalid_argument("AsymptoticLimits: Rule must be either 'CLs' or 'Pmu'");
```

Comment:
Do we want to completely remove the CLsplusb from the user interface all at once, or still allow for backwards compatibility for a little while? Its a minor change, so maybe removing it altogether, as done here, is fine?